Dockerfile

```diff
@@ -1,61 +1,72 @@
-FROM fedora
+FROM alpine:3.5
 
-MAINTAINER Guillaume Scheibel <guillaume.scheibel@gmail.com>
+LABEL MAINTAINER="Guillaume Scheibel <guillaume.scheibel@gmail.com>"
+LABEL MAINTAINER="Damien DUPORTAL <damien.duportal@gmail.com>"
 
-ENV JAVA_HOME /jdk1.8.0_112
-ENV PATH $PATH:$JAVA_HOME/bin:/fopub/bin
-ENV BACKENDS /asciidoctor-backends
-ENV GVM_AUTO_ANSWER true
-ENV ASCIIDOCTOR_VERSION "1.5.5"
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm \
+  PATH=${PATH}:${JAVA_HOME}/bin:/fopub/bin \
+  BACKENDS=/asciidoctor-backends \
```

Comment:
Now that we've made separate gems for reveal.js and bespoke.js, I'm strongly considering dropping the backends from this image. If people disagree, I'm willing to change my mind.